### PR TITLE
Make project v2 ready

### PR DESCRIPTION
**Pull request by nicolas-actyx**
This PR makes this repository ready for v2. The tutorial is changed respectively in this PR: https://github.com/Actyx/Cosmos/pull/6559

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,17 @@
 {
-  "name": "actyx-advanced-tutorial-example-app",
+  "name": "actyx-andvanced-tutorial",
```
Comment (Alexander89) [inline package.json:2]: we should call this advanced example. It does not have any part of a tutorial and is build as example

File: src/dashboard/App.tsx [hunk 2]
```diff
@@ -1,20 +1,39 @@
...
 
 export const App = (): JSX.Element => (
-  <div className="main">
-    <div style={{ flex: '1' }}>
-      <Typography variant="distance">Machines</Typography>
-      <Machines />
+  <>
+    <Toolbar variant="dark">
+      <div style={{ width: '32px', marginRight: '24px', marginLeft: '24px', marginTop: '6px' }}>
+        <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 256 256">
```
Comment (Alexander89) [inline src/dashboard/App.tsx:13]: I think it is better to read when the Logo is in another file

File: src/wago-connector/index.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ import { Fish } from '@actyx/pond'
 // define default settings use during the local development process when
 // when the app is not running in an ActyxOS runtime
 const defaultSettings = {
-  name: 'Machine2',
+  name: 'Wago Connector',
```
Comment (Alexander89) [inline src/wago-connector/index.ts:19]: What's about the `AX_APP_SETTINGS`, isn't that a part that needs to be changed?

**Review by Alexander89 — COMMENTED** [review]
Thanks for making the UI nicer. I think this is not a tutorial. you are not learning something. it is only an example where you can spy up some good.

File: package.json (hunk 1 above)
Comment (nicolas-actyx) [reply package.json:2]: done

File: src/wago-connector/index.ts (hunk 3 above)
Comment (nicolas-actyx) [reply src/wago-connector/index.ts:19]: I removed app settings for this example as we will not support them in v2.

**Pull request merged**